src/Components/ChallengeMenu.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+// import Axios from 'axios';
+
+
+class ChallengeMenu extends React.Component {
+   // constructor(props) {
+   //    super(props);
+
+   // }
+
+
+   render() {
```

Comment:
What you've done here is pretty slick, seems like you're handling the rendering of this menu well.